configure.ac

```diff
@@ -927,6 +927,12 @@ AS_IF([test "x$enable_slingshot" = xyes],[
         [AC_MSG_ERROR([libcxi or its headers not found])])
 ])
 
+AC_ARG_ENABLE([slingshot_switch],
```

Comment:
As far as I can tell, there are no external software dependencies in this sampler. So maybe we don't need to make compilation conditional?

Reply:
@valleydlr We want the ability to optionally not build for those wanting to explicitly limit the build